packages/blockchain-wallet-v4-frontend/src/modals/BuySell/model.tsx

```diff
@@ -237,15 +227,15 @@ export const getPaymentMethodDetails = ({
 }) => {
   switch (order.paymentType) {
     case BSPaymentTypes.PAYMENT_CARD:
-      return `${cardDetails?.card?.type || ''} ${cardDetails?.card?.number || ''}`
+      return `${cardDetails?.card?.type || ''} ***${cardDetails?.card?.number || ''}`
     case BSPaymentTypes.BANK_TRANSFER:
       const defaultBankInfo = {
         accountNumber: '',
         bankAccountType: '',
         bankName: 'Bank Transfer'
       }
       const d = (bankAccount && bankAccount.details) || defaultBankInfo
-      return `${d.bankAccountType?.toLowerCase() || ''} ${d.accountNumber || ''}`
+      return `${d.bankName || ''} ****${d.accountNumber || ''}`
```

Comment:
I know it wasn't part of this PR but could `d` be changed to something more explicit? 
In the same vein, the `defaultBankInfo` could be moved to some place where it is created once, instead of every time `getPaymentMethodDetails` is called.